mne/time_frequency/tests/test_tfr.py

```diff
@@ -204,18 +205,22 @@ def test_time_frequency():
     assert_raises(ValueError, cwt, data[0, :, :], Ws, mode='foo')
     for use_fft in [True, False]:
         for mode in ['same', 'valid', 'full']:
-            # XXX JRK: full wavelet decomposition needs to be implemented
-            if (not use_fft) and mode == 'full':
-                assert_raises(ValueError, cwt, data[0, :, :], Ws,
-                              use_fft=use_fft, mode=mode)
-                continue
-            cwt(data[0, :, :], Ws, use_fft=use_fft, mode=mode)
+            cwt(data[0], Ws, use_fft=use_fft, mode=mode)
 
     # Test decim parameter checks
     assert_raises(TypeError, tfr_morlet, epochs, freqs=freqs,
                   n_cycles=n_cycles, use_fft=True, return_itc=True,
                   decim='decim')
 
+    # When convolving in time, wavelets must not be shorter than the data
```

Comment:
You probably mean "longer than the data"? (for example you take data that is shorter than the wavelet here)